projects/schematics/src/add-spartacus/index.ts

```diff
@@ -171,14 +171,18 @@ function prepareSiteContextConfig(options: SpartacusOptions): string {
   return context;
 }
 
-function getStorefrontConfig(options: SpartacusOptions): string {
-  const baseUrlPart = `\n          baseUrl: '${options.baseUrl}',`;
+function createStorefrontConfig(options: SpartacusOptions): string {
```

Comment:
Can you add a comment to describe what the function is doing?

Reply:
Done